src/App.tsx

```diff
@@ -1,222 +1,107 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import cn from 'classnames';
+import { getTodos } from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+import { Todo } from './types/Todo';
+import { FilterCondition } from './types/FilterCondition';
+import { Header } from './components/Main/Header';
+import { TodoList } from './components/Main/TodoList';
+import { Footer } from './components/Main/Footer';
 
 export const App: React.FC = () => {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
+  const [todosList, setTodosList] = useState<Todo[]>([]);
+  const [isError, setIsError] = useState(false);
+  const [errorText, setErrorText] = useState('');
+  const [filterCondition, setFilterCondition] = useState(FilterCondition.ALL);
+
+  if (isError) {
+    setTimeout(() => setIsError(false), 3000);
+  }
+
+  const getSelected = (allTodos: Todo[]): Todo[] => {
+    return allTodos.filter(item => {
+      switch (filterCondition) {
+        case FilterCondition.ALL:
+          return true;
+
+        case FilterCondition.COMPLETED:
+          return item.completed === true;
+
+        case FilterCondition.ACTIVE:
+          return item.completed === false;
+
+        default: return true;
+      }
+    });
+  };
 
   useEffect(() => {
     // focus the element with `ref={newTodoField}`
     if (newTodoField.current) {
       newTodoField.current.focus();
     }
-  }, []);
+
+    if (user) {
+      getTodos(user?.id)
+        .then(getSelected)
+        .then(setTodosList)
+        .catch(() => {
+          setTodosList([]);
+          setIsError(true);
+          setErrorText('Unable to upload a todo-list');
```

Comment:
Good practice to use ENUM for error messages
```suggestion
          setErrorText(TodoErrors.UnableUpload);
```